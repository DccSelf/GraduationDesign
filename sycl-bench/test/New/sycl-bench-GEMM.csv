# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.696606,0.678821,0.646122,"0.646122 0.647600 0.647687 0.650150 0.655388 0.678821 0.740643 0.742287 0.744378 0.812989",0.059038,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,26.384002,26.531063,25.457767,"25.457767 25.533719 25.682499 25.831851 26.348758 26.531063 26.726162 27.003357 27.314187 27.410656",0.731868,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.571634,0.570801,0.566731,"0.566731 0.567310 0.567780 0.568283 0.568451 0.570801 0.571628 0.572716 0.581245 0.581390",0.005461,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,25.282692,25.476339,22.513642,"22.513642 24.881462 25.094543 25.353318 25.471786 25.476339 25.571750 26.036485 26.092643 26.334950",1.072758,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.649487,0.649259,0.645686,"0.645686 0.645833 0.647192 0.647891 0.648112 0.649259 0.649462 0.650461 0.654008 0.656965",0.003573,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,25.740694,26.133927,23.430072,"23.430072 25.300405 25.387373 25.618188 26.081137 26.133927 26.219651 26.258384 26.403654 26.574147",0.917410,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.617793,0.609915,0.601518,"0.601518 0.601886 0.602026 0.602304 0.608892 0.609915 0.610458 0.610918 0.611338 0.718675",0.035696,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,26.622822,26.832188,25.662076,"25.662076 25.663833 26.086216 26.279199 26.462961 26.832188 26.863098 27.099130 27.430013 27.849502",0.727172,N/A,UNKNOWN,N/A
