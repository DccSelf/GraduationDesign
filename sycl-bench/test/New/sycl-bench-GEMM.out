Using test profile: default


##################################################
Processing gemm-nd_range
##################################################
0.0 10800.0
__________________________________________________

gemm-nd_range --num-runs=10 --output=./sycl-bench-GEMM.csv --size=512 --local=256
==> Benchmark run finished in 13.065159665420651 s
13.065159665420651 10800.0
__________________________________________________

gemm-nd_range --num-runs=10 --output=./sycl-bench-GEMM.csv --size=1024 --local=256
==> Benchmark run finished in 505.3761916188523 s


##################################################
Processing gemm-single_task
##################################################
0.0 10800.0
__________________________________________________

gemm-single_task --num-runs=10 --output=./sycl-bench-GEMM.csv --size=512 --local=256
==> Benchmark run finished in 11.30816741194576 s
11.30816741194576 10800.0
__________________________________________________

gemm-single_task --num-runs=10 --output=./sycl-bench-GEMM.csv --size=1024 --local=256
==> Benchmark run finished in 499.12416717689484 s


##################################################
Processing gemm
##################################################
0.0 10800.0
__________________________________________________

gemm --num-runs=10 --output=./sycl-bench-GEMM.csv --size=512 --local=256
==> Benchmark run finished in 12.061227031983435 s
12.061227031983435 10800.0
__________________________________________________

gemm --num-runs=10 --output=./sycl-bench-GEMM.csv --size=1024 --local=256
==> Benchmark run finished in 505.0622750967741 s


##################################################
Processing gemm-hierarchical
##################################################
0.0 10800.0
__________________________________________________

gemm-hierarchical --num-runs=10 --output=./sycl-bench-GEMM.csv --size=512 --local=256
==> Benchmark run finished in 12.177913499996066 s
12.177913499996066 10800.0
__________________________________________________

gemm-hierarchical --num-runs=10 --output=./sycl-bench-GEMM.csv --size=1024 --local=256
==> Benchmark run finished in 516.756975511089 s
All benchmarks were executed successfully
